scripts/start_fabric.sh

```diff
@@ -44,12 +44,12 @@ if [[ ! $(docker ps --format '{{.Names}}' |grep "peer*") ]]; then
     # to access internet. Passing proxy settings for cli.
     docker rm -f cli
     docker run -dit --name cli --network minifab -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/vars:/vars \
--v $(pwd)/vars/chaincode:/go/src/github.com/chaincode \
+-v $(pwd)/vars/chaincode:/opt/gopath/src/github.com/chaincode \
 -e "http_proxy=$http_proxy" -e "https_proxy=$https_proxy" \
 -e "no_proxy=$no_proxy,localhost,orderer3.example.com,orderer2.example.com,orderer1.example.com,peer2.org1.example.com,peer1.org1.example.com,peer2.org0.example.com,peer1.org0.example.com" \
 -e "HTTP_PROXY=$http_proxy" -e "HTTPS_PROXY=$https_proxy" \
 -e "NO_PROXY=$no_proxy,localhost,orderer3.example.com,orderer2.example.com,orderer1.example.com,peer2.org1.example.com,peer1.org1.example.com,peer2.org0.example.com,peer1.org0.example.com" \
-hyperledger/fabric-tools:2.0
+hyperledger/fabric-tools:1.4.4
```

Comment:
You could use a FABRIC_VERSION variable and set it once and use it at multiple places in this script.

Reply:
Taken care of this.